tests/skyserve/spot/base_ondemand_fallback.yaml

```diff
@@ -6,6 +6,7 @@ service:
     min_replicas: 2
     max_replicas: 3
     base_ondemand_fallback_replicas: 1
+    target_qps_per_replica: 1
```

Comment:
Should this be 1 or larger value? We don't expect it to scale up during the test.

Reply:
SG